.orchestra/config/components/nanobind.yml

```diff
@@ -0,0 +1,34 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create-component.lib.yml", "single_build_component")
+#@ load("/lib/shell.lib.yml", "expand_args")
+#@ load("/lib/cmake.lib.yml", "cmake", "cmdline_cmake_base_configuration")
+#@ load("/lib/optimization-flavors.lib.yml", "typical_cmake_flavors")
+
+#@yaml/text-templated-strings
+---
+#@ def args():
+license: LICENSE
+repository: nanobind
+configure: |
+  git -C "$SOURCE_DIR" submodule update --init --recursive
```

Comment:
This introduces non-determinism :thinking: 
I'd hardcode the hash for the time being.

Reply:
Due to changes in the `nanobind` repo the previous comment no longer applies.
I'm not sure what kind of non-determinism there is, git hardcodes the submodule's `sha`, there is no way (for better or worse) to have non-determinism.

Reply:
Ah OK, thanks for the clarification.

This still doesn't fit super well here, this should likely be part of the `CloneAction.script` in `revng-orchestra`.

Either move it there, or at least leave a TODO about this.

Reply:
Added a TODO